packages/dev/core/src/Meshes/Compression/dracoCompressionWorker.ts

```diff
@@ -128,7 +128,8 @@ export function decodeMesh(
             const ptr = decoderModule._malloc(byteLength);
             try {
                 decoder.GetAttributeDataArrayForAllPoints(geometry, attribute, dataType, byteLength, ptr);
-                const data = new info.typedArrayConstructor(info.heap.buffer, ptr, numValues);
+                // this cast seems to be needed because of an issue with typescript, as all constructors do have the ptr and numValues arguments.
+                const data = new (info.typedArrayConstructor as Float32ArrayConstructor)(info.heap.buffer, ptr, numValues);

```

Comment:
All of the similar constuctors across these different types have the same types for constructor params, except the one overload being used here, where the buffer is the `TArrayBuffer`. I think because it is a type union, they all have 5 identical constructors, but the 6th is not the same between them, so it is not available in this context. Not sure how to fix this though.

Reply:
Wasn't sure how to fix it as well. This is why I simply type-casted to the first one.